ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsCollector.scala

```diff
@@ -46,6 +46,7 @@ class MetricsCollector[T](exposedMetrics: Option[ExposedMetrics[T]], clock: Cloc
   import MetricsCollector.Message._
   import MetricsCollector.Response._
 
+  @scala.annotation.nowarn("msg=.*is unchecked since it is eliminated by erasure")
```

Comment:
I would like to fix it but I don't know yet how I can fix it correctly here without causing big cascades of changes.

Reply:
cc @mziolekda @kamil-da

Reply:
Sorry, I missed this one.
I'm fine with muting this warning. I'll probably need to revisit this piece of code.